cpp/src/arrow/filesystem/s3fs.cc

```diff
@@ -1293,12 +1295,14 @@ std::shared_ptr<const KeyValueMetadata> GetObjectMetadata(const ObjectResult& re
 
 template <typename ObjectRequest>
 struct ObjectMetadataSetter {
+  static constexpr std::string_view kContentTypeKey = "Content-Type";
```

Comment:
"Content-Type" is still used as a literal above. Should we move this at the top-level and use it everywhere?
Or, conversely, just undo this, since the "Content-Type" literal is unlikely to change value...

Reply:
I reverted the change to move it to a constant and rather have it inline now everywhere again.